tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -453,6 +453,16 @@ const {initialState, reducers: namespaceContextedReducer} =
       settings: METRICS_SETTINGS_DEFAULT,
       settingOverrides: {},
       visibleCardMap: new Map<ElementId, CardId>(),
+      previousCardInteractions: {
```

Comment:
This is fine. However, I would prefer to add this state into the reducer when it is starting to be used.

Reply:
I would need to define the state property as optional in order to make typescript happy and I would rather not do that. It's also inconsistent with the way we define other defaults.